src/libs/rover/image.cpp

```diff
@@ -71,7 +71,7 @@ template<typename FloatType>
 void
 Image<FloatType>::normalize_handle(vtkm::cont::ArrayHandle<FloatType> &handle, bool invert)
 {
-
+  // TODO: Surely we can do better than "name meaningless"
   vtkm::cont::Field as_field("name meaningless",
```

Comment:
We should ask ourselves what this is used for. Is the name going to ever be used anywhere?